src/main/resources/config/liquibase/master.xml

```diff
@@ -17,6 +17,7 @@
     <include file="classpath:config/liquibase/changelog/20240617001400_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240612123000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240614140000_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20240701134000_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
I do not see any changeling that adds the new attribute to the programming exercise